### DISCUSSION TITLE
May I ask how to specify the default installation path? I did not see the relevant parameters under the help command

### DISCUSSION THREAD

**Discussion by z582986307** [microsoft/winget-cli#3552] [Configuration]
May I ask how to specify the default installation path? I did not see the relevant parameters under the help command

**Comment by Trenly** [comment]
Currently there is not an ability to set the default installation path for all packages. There are [Settings](https://aka.ms/winget-settings) for the default locations for portable packages and for packages that require an install location to be passed in, but there is not an ability to set it for all packages. There are several complications with this, as publishers don't always write their applications to support custom install locations, don't always respect the install locations that have been passed in, or cases where the way to set the install location simply isn't known.

Some related issues -
* #2849
* #2810
* #201
